<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyServer</class>
 <widget class="QMainWindow" name="MyServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyServer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>511</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_ListenPort">
         <property name="text">
          <string>监听端口号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ListenPort"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Listen">
         <property name="text">
          <string>开始监听</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_SendMessage"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_SendMessage">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_recvfile">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>接收文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SaveFilePath">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
